pkg/config/validation/validation.go

```diff
@@ -468,8 +468,8 @@ var ValidateGateway = registerValidateFunc("ValidateGateway",
 	func(cfg config.Config) (Warning, error) {
 		name := cfg.Name
 		v := Validation{}
-		// Gateway name must conform to the DNS label format (no dots)
-		if !labels.IsDNS1123Label(name) {
+		// Gateway name must conform to the DNS sub domain format.
+		if !labels.IsDNS1123SubDomain(name) {
```

Comment:
We can just remove the name validation here, kube-apiserver will validate it.

Reply:
Agree 👍🏻.